recipe/meta.yaml

```diff
@@ -7,19 +7,25 @@ package:
   version: {{ gfortran_version|join('.') }}
 
 build:
-  number: 2
+  number: 3
   skip: True  # [not osx]
   run_exports:
     strong:
       - libgfortran >={{ libgfortran_version|join('.') }},<{{ max_libgfortran_version }}
 
 requirements:
   host:
+    - clang
     - ld64
-  run:
+    - cctools
     - gfortran_impl_osx-64 =={{ gfortran_version|join('.') }}
     - libgfortran >={{ libgfortran_version|join('.') }},<{{ max_libgfortran_version }}
+  run:
+    - clang
```

Comment:
Should this use the jinja2 compiler syntax?

Reply:
No need to run the activate stuff. `clang` is only needed as an assembler.